### ISSUE TITLE
confirm()  delete of account suggestion

### ISSUE THREAD

**Issue by mrmccormack** [sahat/hackathon-starter#754]
I'm enjoying this fine starter template.
Here is a small suggestion that I think everyone would want.

**File: profile.pug**

Original:
```
line:  69
    form.form-horizontal(action='/account/delete', method='POST')
```
Suggestion with confirm()
```
    form.form-horizontal(action='/account/delete', method='POST', onsubmit="return confirm('Are you sure you want to delete your account?');")

```

**Comment by albertpak** [comment]
👍

**Comment by niallobrien** [comment]
@mrmccormack Can you submit a PR for this please? Thanks.

**Comment by mrmccormack** [comment]
Sure... I did a Pull request at:
https://github.com/mrmccormack/hackathon-starter/pull/1/commits
but it doesn't show up here yet?
Hope that helps.
Rob

**Comment by niallobrien** [comment]
@mrmccormack you'll need to open a PR on this repo requesting to merge the changes from your fork. Thanks.

**Comment by YasharF** [comment]
PR: https://github.com/sahat/hackathon-starter/pull/797

**Comment by YasharF** [comment]
Fixed in v4.4.0 release
